src/util/scatterplotHelpers.js

```diff
@@ -97,10 +97,14 @@ export function getFirstMatchingScatterVariable(options, tryTheseFirst, notThisV
  */
 export function addScatterAxisInfo(scatterVariables, axis, controls, tree, metadata) {
   const axisVar = scatterVariables[axis];
+  console.log("addScatterAxisInfo::axisVar", axis, axisVar)
```

Comment:
intentional? (asking because the linting is complaining.)

Reply:
Ahh. Intentionally added, unintentionally neglected to remove.

Reply:
That's not why CI failed here (that's due to a 6-hour `npm ci` call timing out the action), but I agree with #1839 in that it should cause CI to fail.

Reply:
[dropped in force push](https://github.com/nextstrain/auspice/compare/681951aa9e7793db220cd3c9866a203a2ffcdac5..f5e388ef35cb700fafb5377be4f758d86a188f67)